.github/workflows/check-build-test.yml

```diff
@@ -0,0 +1,220 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches: [ master ]
+    tags-ignore: [ v.* ]
+  schedule:
+    - cron:  '0 0 1 * *' # monthly
```

Comment:
What's you thinking here?

Reply:
I thought there was a monthly cron on Travis, but that's just the default line item to add a cron.

There's a line in `build.sbt` that toggles `Nightly` when the travis event type is `cron`, but if we don't have a travis cron then I don't know its purpose.

Reply:
We used to have Akka 2.6 builds as a Travis cron job.

Reply:
Ok. I'll remove this cron.